src/mbgl/gl/debugging.cpp

```diff
@@ -190,5 +189,3 @@ group::~group() {
 } // namespace debugging
 } // namespace gl
 } // namespace mbgl
-
-#endif
```

Comment:
Removing those include guards means that the symbol `mbgl::gl::debugging::group` is defined (but not declared), while we shouldn't even declare it when `NDEBUG` is set.

It also means that we attempt to load the extensions `GL_KHR_debug` and `GL_EXT_debug_marker` even though we are not using them.